src/main/java/duke/exception/EmptyTextException.java

```diff
@@ -0,0 +1,28 @@
+package duke.exception;
+
+/**
+ * Exception for lack of description of certain things
+ */
+public class EmptyTextException extends DukeException {
+    private String field;
```

Comment:
I am not sure if I fully understand this field. Do you mind clarifying it by giving it a more descriptive name?